ui/sdk/src/hamilton_sdk/api/constants.py

```diff
@@ -1,2 +1,2 @@
 HAMILTON_API_URL = "http://localhost:8241"
-HAMILTON_UI_URL = "http://localhost:8242"
+HAMILTON_UI_URL = "http://localhost:8241"  # optimizing for mini-mode
```

Comment:
err -- docs need to be updated then for use with docker then right?

Reply:
or there needs to be a way to know which URL to use that the SDK can figure out

Reply:
See commit notes, this was the quickest change but yeah we can update docs

Reply:
side note: `127.0.0.1` should be preferred to `localhost`  [ref](https://www.youtube.com/watch?v=98SYTvNw1kw)

Reply:
Hmm I'm generally against using a hardcoded value for the purpose of small optimizations, although there are other reasons... Good video.